pkg/testsuites/standard_suites.go

```diff
@@ -269,6 +269,21 @@ var staticSuites = []ginkgo.TestSuite{
 		Count:       12,
 		TestTimeout: 20 * time.Minute,
 	},
+	{
+		Name: "openshift/network/live-migration",
+		Description: templates.LongDesc(`
+		This test suite performs CNI live migration either from SDN to OVN-Kubernetes or OVN-Kubernetes to SDN.
+		`),
+		Matches: func(name string) bool {
+			if isDisabled(name) {
+				return false
+			}
+			return strings.Contains(name, "[Suite:openshift/network/live-migration")
+		},
+		Count:                      1,
+		TestTimeout:                4 * time.Hour,
+		ClusterStabilityDuringTest: ginkgo.Stable,
```

Comment:
I know you changed this because of the watchdog alert.  We have https://github.com/openshift/origin/pull/28518 up to fix that.  Wanted to check to see if switching the network is expected to be disruptive or not.

Reply:
@neisw I believe upgrade uses this Stability criteria aka `ginkgo.Stable` and we want to equal this. I was wrong to put it as `ginkgo.Disruptive` before. Thanks!